test/large_pos/tview.expected.out.new

```diff
@@ -0,0 +1,48 @@
+ 10000000001          10000000021         10000000041         10000000061       
```

Comment:
This file will be automatically removed after running the test harness, because of its extension matching a clean-up rule in the [Makefile](https://github.com/samtools/samtools/blob/796cf226ff23771c754e9976fda85efa20414737/Makefile#L317).

Reply:
It shouldn't have been added.  It's now been removed.